Migrate user submitting returns test

Cruikshanks opened the pull request:
This migrates [user-submitting-returns.spec.js](https://github.com/DEFRA/water-abstraction-ui/blob/main/cypress/integration/external/user-submitting-returns.spec.js) in the old repo.

When we checked the existing spec we found it contained 5 tests!

- submit a return metered readings return
- submit a null return
- submit a return by volumes
- submit returns measured in gallons
- submit returns measured in megalitres

To make that clearer we've split them as part of the migration.

timstone commented on features/step_definitions/manage_licences_steps.rb:
```diff
@@ -13,6 +13,7 @@
   @front_app.manage_licences_page.add_user_button.click
+  @front_app.manage_give_access_page.add_user_button.click
   expect(@front_app.manage_give_access_page.heading).to have_text("Give access to view your licences")
   @geoff_email = @front_app.manage_give_access_page.generate_email.to_s
```
As discussed either reference Geoff as a user persona in the feature or change it to something more meaningful for an agent

andrewhick commented:
Pushed all changes relating to notifications

andrewhick commented:
Merging based on discussion with Tim - all issues and conflicts are resolved.

Cruikshanks committed:
Migrate user submitting returns test

This migrates [user-submitting-returns.spec.js](https://github.com/DEFRA/water-abstraction-ui/blob/main/cypress/integration/external/user-submitting-returns.spec.js) in the old repo.

When we checked the existing spec we found it contained 5 tests!

- submit a return metered readings return
- submit a null return
- submit a return by volumes
- submit returns measured in gallons
- submit returns measured in mega litres

To make that clearer we've split them as part of the migration.

Cruikshanks committed:
Migrate the tests
